### ISSUE TITLE
ClusterMgr can't detect upgraded Gluu Server 

### ISSUE THREAD

**Issue by mzico** [GluuFederation/cluster-mgr#268]
I upgraded my 3.1.3 to 3.1.6. 
ClusterMgr still detecting 3.1.3 instead of correct 3.1.6. Screenshot attached. 

And, I already used 'clear_prop.py' to generate a simplified version of setup.properties.last and added '3.1.6' there as Gluu Server version but still ClusterMgr detecting it as 3.1.3. 
Note that, my container name is still 'gluu-server-3.1.3' [ i.e. service gluu-server-3.1.3 login ] 
![Capture](https://user-images.githubusercontent.com/7156097/63046913-f4a0dc00-be98-11e9-929e-9456662d2cc9.PNG)

**Comment by mzico** [comment]
clustermgr log saying... 

```
[2019-08-15 11:02:47,087: INFO/ForkPoolWorker-1] @ip-172-31-23-187.us-west-2.compute.internal> listing directory: /opt
[2019-08-15 11:02:47,132: INFO/ForkPoolWorker-1] @ip-172-31-23-187.us-west-2.compute.internal> list of directory /opt: [u'gluu-server-3.1.3', u'gluu-server-3.1.3.tar', u'.#gluu-server-3.1.3.lck']
```

That means, ClusterMgr is trying to detect the version of Gluu-Server from the name of 'container'. It should read from 'setup.properties.last' `oxVersion` instead.

**Comment by mbaser** [comment]
Currently cluster manager uses `/opt/gluu-server-x.y.z` for detecting server version.

**Comment by willow9886** [comment]
Can we find a better way to detect? In Gluu 4.0, we're removing the version from the package.  Also, the above method causes issues for upgraded environments, e.g. currently, an upgraded Gluu 3.1.6 server from Gluu 3.1.3 would still have   `./opt/gluu-server-3.1.3`

**Comment by willow9886** [comment]
https://github.com/GluuFederation/support-docs/blob/master/howto/upgrade/Change_container_name_after_upgrade.md

**Comment by mbaser** [comment]
The above instructions are implemented in commit 1970142a1ac98e86a21b97f2c25cb398f32a1ddc

**Comment by mbaser** [comment]
Tested, it is working.
